lib/root_controller.js

```diff
@@ -0,0 +1,174 @@
+var Tessel = require('./tessel/tessel');
+var logs = require('./logs');
+var debug = require('debug')('controller');
+
+// create an object for t2 root steps for beeing more handy about unit testing of the T2 root command itself.
+var controller = {};
+controller.ssh = {
```

Comment:
I learned to use module pattern where ever possible. Maybe it would be better to locate the file on a better place? I think the controller.js would grow much more over time - while developing and writing tests I noticed I am losing overview due to much additional things I put into code (what is away now - but it pointed me on to create a module). Of cause it isn't a reusable module but I also saw there are different parts in the t2-cli are done that way - so I thought I should do it the same way.
If no more comments on it, I will simple merge it back into controller.js